packages/design-system-tailwind-preset/src/colors.test.ts

```diff
@@ -48,9 +48,9 @@ describe('Color Preset', () => {
     const usedSet = new Set(usedVariables);
     const ignoredSet = new Set(ignoreList);
 
-    // Identify design token variables that are neither used nor ignored
+    // Filter out variables that are either used or explicitly ignored
     const unusedVariables = Array.from(designTokens).filter(
-      (varName) => !usedSet.has(varName) && !ignoredSet.has(varName),
+      (varName) => !usedSet.has(varName) && !ignoredSet.has(varName), // eslint-disable-line jest/no-conditional-in-test -- Legitimate data processing for test setup
```

Comment:
The ESLint rule jest/no-conditional-in-test is designed to prevent non-deterministic test behavior, but in these specific cases, the conditional logic is:

- Deterministic and necessary: The filter operations process data consistently based on predefined sets
- Essential for proper mocking: The theme function conditionals simulate Tailwind's actual API behavior

This approach maintains code readability and functionality while properly documenting why these specific conditionals are appropriate exceptions to the general rule.